docs/yaml/functions/executable.yaml

```diff
@@ -10,6 +10,9 @@ description: |
 
   The returned object also has methods that are documented in [[@exe]].
 
+  *Since 1.3.0* executable names can be the same across multiple targets as
+  long as they each have a different `name_suffix`.
```

Comment:
Is there some way we can detect this case at setup time and emit a FeatureNew?

Reply:
Ah good idea. I think it should be possible. I'll try to come up with something.

Reply:
Okay I found the spot to detect this. Wasn't too bad.

Reply:
Fantastic! <3